src/test/java/org/xstefank/check/additional/OneCommitOnlyCheckTest.java

```diff
@@ -0,0 +1,41 @@
+package org.xstefank.check.additional;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.xstefank.TestUtils;
+
+import java.io.IOException;
+
+public class OneCommitOnlyCheckTest {
+
+    private OneCommitOnlyCheck oneCommitOnlyCheck;
+
+    @Before
+    public void before() {
+        oneCommitOnlyCheck = new OneCommitOnlyCheck();
+    }
+
+    @Test
+    public void testIfPayloadHasOnlyOneCommit() {
+        Assert.assertNull("Invalid commits count", oneCommitOnlyCheck.check(TestUtils.TEST_PAYLOAD));
+    }
+
+    @Test
+    public void testIfPayloadHasMoreCommits() {
+        Assert.assertNotNull("Commits count should not be valid", oneCommitOnlyCheck.check(TestUtils.BAD_TEST_PAYLOAD));
+    }
+
+    @Test(expected = NullPointerException.class)
+    public void testIfNullParameterThrowsException() {
```

Comment:
It's visible on line 31 that you expect this test to throw exception, no need to repeat it here in name